engine/resource/src/dmsdk/resource/resource.hpp

```diff
@@ -0,0 +1,227 @@
+// Copyright 2020-2024 The Defold Foundation
```

Comment:
Manually created/edited C++ api, for those things that are harder to generate (at least some of it)